### DISCUSSION TITLE
Bloquear acesso ao repositório de máquinas diferentes

### DISCUSSION THREAD

**Discussion by daylane** [community/community#39322] [New to GitHub]
Alguém saberia me informar, se existe a possibilidade de bloquear outras máquinas de acessaram/clonarem um repositório? 
Sei que existe a opção de deixar o repositório privado mas caso seja um colaborador que utiliza o repositório no âmbito de trabalho e queira levar pra casa, teria como bloquear isso?
Ou pelo menos fazer verificação do local/endereço/maquina que foi commitado?

**Comment by jfullstackdev** [comment]
maybe this ? 

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/about-protected-branches

**Comment by queenofcorgis** [comment]
Hi @Daylane, thanks for participating!

We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
